api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+          type: integer
+          format: int32
+          required: false
+        - name: batch_keep_alive_limit
+          in: query
+          description: Maximum number of keep-alive messages to get in a row before closing the connection. Unlimited by default.
+          type: integer
+          format: int32
+          required: false
+      responses:
+        '200':
+          description: |
+            Starts streaming to the client.
+            Stream format is a continuous series of
+            `SimpleStreamEvent`s separated by `\n`
+          schema:
+            $ref: '#/definitions/SimpleStreamEvent'
+        '400':
+          description: Bad syntax
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          schema:
+            $ref: '#/definitions/Problem'
+    post:
+      description: |
+        Post a one or more `Event`s to a given `Topic`.
+        The partition is implicitly controlled by the partitioning_key attribute of the event. Nakadi guarantees that
+        all events with the same partitioning_key end up in the same partition.
+      parameters:
+        - name: topic
+          in: path
+          description: Topic name where to send events to
+          type: string
+          required: true
+        - name: event
+          in: body
+          description: The body contains an array of events being sent.
+          required: true
+          schema:
+            type: array
+            description: An array of events to be pushed to Nakadi
+            items:
+              $ref: '#/definitions/Event'
+        - name: Content-Encoding
+          in: header
+          required: false
+          description: '"Allow the publisher to gzip compress the payload by setting this header to "gzip". For Example "Content-Encoding: gzip"'
+          type: string
+      responses:
+        '201':
+          description: All events are submitted successfully.
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Not allowed
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+  '/topics/{topic}/partitions':
+    get:
+      tags:
+        - low-level-api
+        - monitoring
+      summary: Lists the `Partition`s for the given topic
```

Comment:
summary: returns all partitions including its current offsets for a given topic.